src/components/IconText/IconText.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import './IconText.scss';
+
+class IconText extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        return (
+            <div className={'icon-text ' + this.props.class} style={{ width: this.props.width }}>
+                {this.props.position === 'left' ? (
+                    <>
```

Comment:
why?

Reply:
should be fixed with `npm run lint:prettier -- --write`